Automattic/WP-Job-Manager
# Restructure main admin menu

Fixes #2586 

### Changes Proposed in this Pull Request

- Restructure menu to match the design as shown on #2586 

### Testing Instructions

Using this branch of the plugin to test:

1. Make sure the menu matches the design
2. Check the icon, order of the items, and labels used in the menu
3. Verify if no icons break with the changes


### Release Notes

* Restructure the main admin menu of the plugin

### Screenshot / Video

![New restructured menu](https://github.com/Automattic/WP-Job-Manager/assets/529864/0fb1ec67-5967-4b9f-9e6d-faf0bea7f3d5)




----

| Plugin build for 941dfb842a4431a09b7ddfa2be6ed935a9339d3d <a href="#"><img width=600></a> |
| ------------------------------------------------------------ |
| 📦 [Download plugin zip](https://wpjobmanager.com/wp-content/uploads/2023/10/wp-job-manager-zip-2612-941dfb84.zip)                       |
| ▶️ [Open in playground](https://wpjobmanager.com/playground/?core=2023/10/2612-941dfb84)             |
